### ISSUE TITLE
Return country results for inexact country searches

### ISSUE THREAD

**Issue by MattBlissett** [gbif/portal16#96]
I think all these searches should return the country result/results:

http://internal.gbif-uat.org/search?q=Congo (Rep. Congo and D.R. Congo)

http://internal.gbif-uat.org/search?q=China (China and Chinese Taipei)

http://internal.gbif-uat.org/search?q=Taiwan

http://internal.gbif-uat.org/search?q=Chinese%20Taipei

http://internal.gbif-uat.org/search?q=USA

It's also not at all clear that only participant countries are returned, is that intentional?

**Comment by MortenHofft** [comment]
I agree that would be nice. Currently we do not really have an API for reliably searching countries and participants based on names. So what we do now is using the directory API. That puts a lot of restraints on usage, but allowed us to do something at least. But for sure that would be an obvious and welcome improvement.

**Comment by MattBlissett** [comment]
I don't know how likely it is that GB attendees will try the website themselves. If they do, the first thing they'll do is search for their country.

**Comment by kcopas** [comment]
It is, but we've only just started talking about it. My only gloss on the notes Tim scratched out for me this morning is to 'manage expectations on country pages'—I think I need to tackle it head-on and say 'This isn't going to work yet, but please don't worry'
